src/registrar/assets/sass/_theme/_pagination.scss

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
@rachidatecs 
Just reviewed the updates.
- Can you remove the period in the "no results" message? Now that we removed the second sentence, we no longer need it for consistency. I think it looks odd.
- When we huddled yesterday, I thought we agreed the "reset" button would only appear if a search had been completed. Can you update it so it displays conditionally? 
- After completing a search and clicking "reset," the counter (below table) still displays the search term. That shouldn't display anymore.